python/cudf/bindings/search.pyx

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+
+# cython: profile=False
+# distutils: language = c++
+# cython: embedsignature = True
+# cython: language_level = 3
+
+# Copyright (c) 2018, NVIDIA CORPORATION.
+
+from cudf.bindings.cudf_cpp cimport *
+from cudf.bindings.cudf_cpp import *
+
+from cudf.bindings.search import *
+
+from cudf.dataframe.column import Column
+
+def search_sorted(column, values, side):
+    """Find indices where elements should be inserted to maintain order
+
+    Parameters
+    ----------
+    column : Column
+        Column to search in
+    values : Column
+        Column of values to search for
+    side : str {‘left’, ‘right’} optional
+        If ‘left’, the index of the first suitable location found is given.
+        If ‘right’, return the last such index
+    """
+    cdef gdf_column *c_column = column_view_from_column(column)
+    cdef gdf_column *c_values = column_view_from_column(values)
```

Comment:
You're leaking both of these

Reply:
Fixed.
BTW, I copied from here https://github.com/rapidsai/cudf/blob/73e084956c093765321869e8046159030215128f/python/cudf/cudf/bindings/stream_compaction.pyx#L68-L88

Reply:
Yea, seems like that's leaking as well, we'll have to resolve.